config/nuttx/spresense/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>firmware</name>
+  <version>0.0.1</version>
+  <description>This is a dummy package to supply firmware build dependencies for spresense.</description>
+  <maintainer email="ingo.luetkebohle@de.bosch.com">Ingo Luetkebohle</maintainer>
```

Comment:
```suggestion
  <maintainer email="Barry.Xu@sony.com">Barry Xu</maintainer>
  <maintainer email="Tomoya.Fujita@sony.com">Tomoya Fujita</maintainer>
```

Reply:
@fujitatomoya  

Thanks for your review comments. I have addressed them at https://github.com/micro-ROS/micro_ros_setup/pull/323/commits/4184ad96f7ec3b1e20465bcc64973798486ee76a.